media/css/mozorg/home/home.scss

```diff
@@ -140,6 +140,20 @@ body {
     }
 }
 
+html[dir="rtl"] .masthead-nav-main {
```

Comment:
Rather than fix this for each page, could this be in `_masthead.scss` instead maybe?